hazelcast/src/main/java/com/hazelcast/map/impl/nearcache/invalidation/BatchNearCacheInvalidation.java

```diff
@@ -48,11 +44,15 @@ public void consumedBy(InvalidationHandler invalidationHandler) {
         invalidationHandler.handle(this);
     }
 
-    public void add(SingleNearCacheInvalidation invalidation) {
+    public void add(Invalidation invalidation) {
```

Comment:
Do we even need this add method? It looks like it is only being used from the `BatchNearCacheInvalidationTest#equals_itself_after_deserialization` and the test could very well construct the `BatchNearCacheInvalidation` with the list. The invalidations field default `EMPTY_LIST` can also then be removed since it is always assigned in the constructor.